src/VolleyManagement.Backend/VolleyManagement.Data.MsSql/Repositories/TeamRepository.cs

```diff
@@ -82,10 +84,16 @@ public void Update(Team updatedEntity)
         /// <param name="teamId">The id of team to remove.</param>
         public void Remove(TeamId teamId)
         {
-            var dalToRemove = new TeamEntity { Id = teamId.Id };
-            _dalTeams.Attach(dalToRemove);
-            _dalTeams.Remove(dalToRemove);
-            _unitOfWork.Commit();
+            var teamtodelete = _dalTeams.Find(teamId.Id);
```

Comment:
Tell what was the reason for this change?

Reply:
Method Attach throwed error that that team exist and can not attach it to context
![image](https://user-images.githubusercontent.com/17709640/39520953-2c4aa1ea-4e15-11e8-8bfb-b70df0358c52.png)

Reply:
I disagree. In production code it works fine - and this is the most important use case